<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserEditDialog</class>
 <widget class="QDialog" name="UserEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelLogo">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resource.qrc">:/res/Images/user_edit.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Редагування профілю користувача</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Login:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEditLogin"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ПІБ:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEditPIB"/>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="checkBoxIsActive">
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Активний</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ролі</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="rolesLayout"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Close|QDialogButtonBox::StandardButton::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
